signal/src/BUILD

```diff
@@ -33,3 +27,15 @@ cc_library(
     srcs = ["window.cc"],
     hdrs = ["window.h"],
 )
+
+cc_library(
```

Comment:
Any particular reason for moving this? Probably doesn't need to be alphabetically sorted, but wasn't sure if there was a reason.

Reply:
I only moved it for alphabetical sorting because suleman preferred that previously when I was trying to land this , will keep in mind for future changes and land as is.